PivotGrid: update Overview (#7530)
DevExpress/devextreme-documentation: concepts/05 UI Components/PivotGrid/02 Overview/10 Key Features.md

[Export PivotGrid data to Excel](/concepts/05%20UI%20Components/PivotGrid/090%20Export/00%20Export.md '/Documentation/Guide/UI_Components/PivotGrid/Export/') in XLSX or CSV format and customize the exported cells.

- **Chart Integration**    
PivotGrid handles large numeric datasets that can be hard to interpret. For a detailed data visualization, [use the Chart component](/concepts/05%20UI%20Components/PivotGrid/150%20Integration%20with%20Chart/10%20Integration%20with%20Chart.md '/Documentation/Guide/UI_Components/PivotGrid/Integration_with_Chart/') that integrates into PivotGrid.

- **Virtual Scrolling**    
If PivotGrid is bound to a large dataset, you can [enable a virtual scrolling mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to optimize data load time and reduce resource consumption.

- **Drill Down**    

> PivotGrid supports the [drill-down operation](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/DrillDown/) that allows you to retrieve individual records used to calculate a specific summary value.

Reviewer: ```suggestion
PivotGrid supports [drill-down operations](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/DrillDown/) that allow you to retrieve individual records for calculating a specific summary value.
```
